ciphers/brute_force_transposition_cipher.py

```diff
@@ -0,0 +1,29 @@
+import math
+
+def decryptMessage(key, message):
```

Comment:
follow consistent conventions for naming a function, either use **snake_case** or use **camelCase**, since the repo is `Python` you should only be using **snake_case**